README.md

```diff
@@ -151,3 +152,33 @@ Use npm modules with node that mimic the same tasks but are system agnostic. Som
 * set environment variable (`ENV_VAR = ...`) -> [`cross-env`](https://www.npmjs.com/package/cross-env)
 
 [↑ back to Table of Contents](#table-of-contents)
+
+## Document your script API
+
+### Why?
+
+* Documentation provides developers with a high level overview to the script, without the need to go through all its code. This makes a module more accessible and easier to use.
+
+* Documentation formalises the API.
+
+* README.md is the de facto standard filename for documentation to be read first. Code repository hosting services (Github, Bitbucket, Gitlab etc) display the contents of the the README's, directly when browsing through source directories. This applies to our module directories as well.
+
+
+### How?
+
+Use markdown syntax for creating simple tables.
+```markdown
+`npm run ...` | Description
+---|---
+task | What it does as a plain human readable description.
+```
+
+An example:
```

Comment:
Maybe change `build`, `build:tags` and `build:min` into a more common setup:

`build` | Compile, bundle and minify all CSS and JS files.
`build:css` | Compile, autoprefix and minify all CSS files to `dist/index.css`.
`build:js` | Compile, bundle and minify all JS files to `dist/index.js`.

And change the description of `test` a bit otherwise it's like commenting the name of a variable. So:

`test` | Run all unit and end-to-end tests.